paasta_tools/marathon_tools.py

```diff
@@ -959,7 +959,10 @@ def get_healthcheck_for_instance(service, instance, service_manifest, random_por
     or (None, None) if no healthcheck
     """
     smartstack_config = load_service_namespace_config(service, instance, soa_dir)
-    mode = service_manifest.get_healthcheck_mode(smartstack_config)
+    try:
+        mode = service_manifest.get_healthcheck_mode(smartstack_config)
```

Comment:
What do you think about having a `get_healthcheck_mode` func in the adhoc config that just returns None?